go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/bitrise-steplib/steps-restore-carthage-cache
 go 1.17
```

Comment:
Slightly unrelated, but it's a oneliner, so why not: the module declaration in the first line is not correct, can you please update it to `module github.com/bitrise-steplib/bitrise-step-restore-carthage-cache/step`. You might need to fix an import in main.go too.

Reply:
done

Reply:
I think you meant 

`module github.com/bitrise-steplib/bitrise-step-restore-carthage-cache` though it seems, correct?